static/app/views/replays/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import withPageFilters from 'sentry/utils/withPageFilters';
+
+const Body = styled('div')`
+  background-color: ${p => p.theme.backgroundSecondary};
+  flex-direction: column;
+  flex: 1;
+`;
+
+// TODO: feature check on 'replays'
+const ReplaysContainer: React.FC = ({children}) => <Body>{children}</Body>;
```

Comment:
Let's do this before merging so that this isn't accessible when visiting the route directly.